img/t/tauros.io.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><path fill="#42CE63" d="M54.13 38.57V90H34.86V38.57h19.26zm-28.88-22.5v41.79H6V16.07h19.25zM83 0v57.86H63.75V0H83z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><path fill="#070707" d="M54.13 38.57V90H34.86V38.57h19.26zm-28.88-22.5v41.79H6V16.07h19.25zM83 0v57.86H63.75V0H83z"/></svg>
```
Although the favicon is green, they don't seem to use that logo anywhere else. I think that it may be better to stick with the logo in `#070707` like on their website.